py3status/modules/battery_level.py

```diff
@@ -83,9 +90,11 @@
 from __future__ import division  # python2 compatibility
 from time import time
 from re import findall
+from glob import iglob
 
 import math
 import subprocess
+import os
```

Comment:
I'd be inclined to do `import os.path` as that is all we want,  this might be a just me sort of thing.